cpp/include/nvtext/jaccard.hpp

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) 2023, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include <cudf/column/column.hpp>
+#include <cudf/strings/strings_column_view.hpp>
+
+namespace nvtext {
+/**
+ * @addtogroup nvtext_jaccard
+ * @{
+ * @file
+ */
+
+/**
+ * @brief Computes the Jaccard similarity between individual rows
+ * in two strings columns
+ *
+ * The similarity is calculated between strings in corresponding rows
+ * such that `output[row] = J(input1[row],input2[row])`.
+ *
+ * The Jaccard index formula is https://en.wikipedia.org/wiki/Jaccard_index
+ * ```
+ *  J = |A ∩ B| / |A ∪ B|
+ *  where |A ∩ B| is number of common values between A and B
+ *  and |x| is the number of unique values in x.
+ * ```
+ *
+ * The computation here compares strings columns by treating each string as text (i.e. sentences,
```

Comment:
From the perspective of an ngram it's probably not useful to have an ngram of n=1, but perhaps it does for Jaccard?

Reply:
Discussed offline, I'm fine leaving this documentation as is.

Reply:
@VibhuJawa is n=1 important for any of your use cases, or do you anticipate it ever being so? If not, I'll resolve this with no action. Otherwise, we should open an issue to track that generalization.

Reply:
Nope, not important to any use case i am aware of.

Reply:
OK cool thanks.